_posts/2019-07-09-ranking-factors.md

```diff
@@ -45,20 +45,22 @@ Note: Sites using the search results API to present our results on their own web
 
 ### Core Ranking Algorithm
 
-Our system is built on Elasticsearch, which itself is built on Apache Lucene. For the first several generations, Elasticsearch used Lucene’s default ranking, the Practical Scoring Function. This Function starts with a basic Boolean match for single terms and adds in TF/IDF and a vector space model. Here are some high level definitions for these technical terms:
+Our system is built on Elasticsearch, which itself is built on Apache Lucene. 
+
+The Practical Scoring Function, the primary ranking algorithm, uses a basic Boolean match for single terms and adds in TF/IDF and a vector space model. Here are some high level definitions for these technical terms:
 
 * **Boolean matches** are the AND / OR / NOT matches you’ve probably heard about.
   * This AND that 
   * This OR that
   * This NOT that
   * This AND (that OR foo) NOT bar
-  * **Note** that while the relevance ranking takes these into account, we do not currently use these operators if entered by a searcher. Support for user-entered Boolean operators is coming in 2019.
+  * **Note:** While the relevance ranking takes these into account, we do not currently use these operators if entered by a searcher.
 * **TF/IDF** means term frequency / inverse document frequency. It counts the number of times a term appears in a document, and compares it to how many documents have that word. It aims to identify documents where the query terms appear frequently, and documents with more rare terms across the whole set of documents will get a higher score. Documents with a lot of common terms appearing in many documents will get a lower score.
-  * They also have tempered the TF/IDF score with a method called **BM25**, which attempts to balance the TF/IDF scores of documents that are very different in length. If there are ten documents containing rare terms, the longest doc with the most instances of the terms would get a much higher score than a short doc with only a few instances of the terms. This makes intuitive sense, but when considered as a full pdf of a report vs the summary of the report, the full report isn't that much more relevant to the query than the summary is. BM25's length 'normalizatin' addresses that issue.
+  * They also have tempered the TF/IDF score with a method called **BM25**, which attempts to balance the TF/IDF scores of documents that are very different in length. If there are ten documents containing rare terms, the longest doc with the most instances of the terms would get a much higher score than a short doc with only a few instances of the terms. This makes intuitive sense, but when considered as a full pdf of a report vs the summary of the report, the full report isn't that much more relevant to the query than the summary is. BM25's length 'normalization' addresses that issue.
```

Comment:
please change "They also have tempered the TF/IDF score" to "Elasticsearch modifies the basic TF/IDF score"

Reply:
And thanks for catching the typo.